packages/universal_profiling_symbolizer/manifest.yml

```
@@ -25,6 +25,27 @@ policy_templates:
           - name: host
             type: text
             default: localhost:8240
+            description: Network address used to serve HTTP endpoint for symbtool
+          - name: verbose
+            type: bool
+            default: false
+            description: Enable debug logging
+          - name: telemetry
+            type: bool
+            default: true
+            description: Enable metrics
+          - name: memory_limit
+            type: text
+            default: ""
+            description: |
+              Set a soft memory limit for the program. The format for Megabytes is "MB", for Mebibytes is "MiB", etc...
+              The default value ("") does not apply any limits
```

`[..] does not apply any limits` is not correct.

```suggestion
              The default value ("") applies a memory limit of 25% of the available memory.
```